_locales/en/messages.json

```diff
@@ -158,10 +158,13 @@
       "description": "Label for a button to accept a new safety number"
     },
     "verify": {
-      "message": "Verify"
+      "message": "Mark as verified"
+    },
+    "unverify": {
+      "message": "Clear verification"
```

Comment:
I think "Mark as not verified" is better than "Clear verification" for consistency with other copy like "You marked your safety number with $name$ as unverified." One verb to rule them all.